<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>942</width>
                <height>531</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
            <item>
                <layout class="QGridLayout" name="gridlayout"/>
            </item>
            <item>
                <widget class="QWidget" name="PlayControls" native="true">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="autoFillBackground">
                        <bool>false</bool>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,0,3">
                        <property name="sizeConstraint">
                            <enum>QLayout::SetDefaultConstraint</enum>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QLineEdit" name="seekValue">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="inputMask">
                                    <string/>
                                </property>
                                <property name="maxLength">
                                    <number>30</number>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="seekButton">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>80</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>80</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>Seek</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_2">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::Minimum</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>10</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="prevFrameButton">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>80</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>80</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>Prev</string>
                                </property>
                                <property name="shortcut">
                                    <string>Left</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="playPauseButton">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="text">
                                    <string>Play</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="nextFrameButton">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>80</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>80</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>Next</string>
                                </property>
                                <property name="shortcut">
                                    <string>Right</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::Minimum</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>10</width>
                                        <height>10</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="resetViewButton">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="text">
                                    <string>Reset View</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_3">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::Minimum</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>10</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QLabel" name="frameNumber">
                                <property name="minimumSize">
                                    <size>
                                        <width>150</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>150</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>0</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_4">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QSlider" name="seekBar">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
